### DISCUSSION TITLE
Texture colours or lighting changing of own accord

### DISCUSSION THREAD

**Discussion by MATTJ0NES** [google/model-viewer#2510] [Q&A]
Hi,
I have a glb model that has red texture jpegs and shows as red when I display in any other 3d viewer. When shown in modelviewer, the model colour is initially orange when it loads. If you wait 10-15 seconds, the model darkens to the red I believe it should initially be. If you wait more it cycles back around from red to orange.

What is causing this to change over time and how can I keep to the red colour it should be?

I've added the model to the glitch below which produces the same issue.

Thanks

#### Live Demo
https://futuristic-autumn-value.glitch.me/

### Version
https://unpkg.com/@google/model-viewer@1.7.2/dist/model-viewer.min.js

### Browser Affected
Chrome v91.0.4472.77
iOS 14.6 Safari

### OS
iOS 14.6
MacOS Big Sur 11.4

**Comment by elalish** [comment]
This is intended behavior for a Physically-based Renderer. Our default IBL has more light on the front than the sides/back, so you're seeing a lighter color on the front. If you wait for the model to turn all the way around, you'll see it comes back to that lighter shade. If you want colors to come through unchanged, you'll need to use the glTF materials_unlit extension. Otherwise, you might be interested in `environment-image="neutral"`, which has more even lighting than our [default](https://modelviewer.dev/examples/lightingandenv/#neutralLighting).

**Reply by MATTJ0NES** [reply]
Thank you. Setting environment-image="neutral" made enough of a difference for me to go with that. Maybe I was misunderstanding how the lighting worked on the rotation as if I drag back to the starting position when it's turned 90 degrees then the colour is darker even though by rotating to it's original position I presume the light is also in the original position. But either way, I'm happy with using the neutral setting so thanks again!

**Reply by Kn5000** [reply]
How do you set environment-image="neutral" ? I can't find it in the drop down menu of "Environment Image". I've also tried to edit the snippet, but when saved, it doesn't take changes into account. Please help! Thanks in advance 

![Capture](https://user-images.githubusercontent.com/89594859/131005598-4e5e69f4-524e-4f64-b1fd-3908dd7afc9a.JPG)

**Reply by elalish** [reply]
Good catch, thanks. I'm working on updating a bunch of Editor UI right now and fixing bugs, so I'll add this to my list.

**Reply by Kn5000** [reply]
Thanks for that. Is there a way to have a try already by editing the snippet? Sorry to push, I'm trying to get a project finalized asap :)

**Reply by elalish** [reply]
I can't  fix an editor bug quite that fast 😂. I'd recommend testing by remixing our Glitch: https://glitch.com/edit/#!/model-viewer
